cg/meta/workflow/rnafusion.py

```diff
@@ -181,6 +181,15 @@ def write_trailblazer_config(self, case_id: str, tower_id: str) -> None:
             file_path=config_path,
         )
 
+    def get_last_tower_id(self, case_id: str) -> int:
```

Comment:
This seems like a general function for any tower analysis. I think it should be in Tower API.

Reply:
agree! thanks